ddtrace/tracer.py

```diff
@@ -630,30 +646,6 @@ def _initialize_span_processors(self):
             ),
         ]  # type: List[SpanProcessor]
 
-    def _check_new_process(self):
-        # type: () -> bool
-        """Checks if the tracer is in a new process (was forked) and performs
-        the necessary updates if it is a new process
-        """
-        pid = getpid()
-        if self._pid == pid:
-            return False
-
-        self._pid = pid
```

Comment:
This line got deleted but we should still need it 🤔

I really hope a test case caught it... if not we should definitely add one